src/palace/manager/sqlalchemy/model/patron.py

```diff
@@ -168,30 +168,36 @@ class Patron(Base, RedisKeyMixin):
     neighborhood: str | None = None
 
     loans: Mapped[list[Loan]] = relationship(
-        "Loan", back_populates="patron", cascade="delete", uselist=True
+        "Loan",
+        back_populates="patron",
+        cascade="delete",
+        uselist=True,
+        passive_deletes=True,
```

Comment:
See SQLAlchemy docs: https://docs.sqlalchemy.org/en/20/orm/cascades.html#passive-deletes

I updated these relationships to set the `passive_deletes` parameter since I added a DB level foreign key cascade.